packages/p/protobuf-cpp/xmake.lua

```diff
@@ -39,6 +39,7 @@ package("protobuf-cpp")
 
     on_install("windows", "linux", "macosx", function (package)
         os.cd("cmake")
+        io.replace("CMakeLists.txt", "set(protobuf_DEBUG_POSTFIX \"d\"", "set(protobuf_DEBUG_POSTFIX \"\"", {plain = true})
```

Comment:
It would be better to link the correct library name (libprotobuff(d)) in debug) than patch CMakeLists

Reply:
reason is the pkgconfig pc file dont has d
If you want to link the correct name, you need two patch files `protobuf-lite.pc.cmake` and `protobuf.pc.cmake`